Makefile

```diff
@@ -399,7 +399,15 @@ dev-azure-nuke: envsubst azure-nuke ## Warning: Destructive! Nuke all Azure reso
 	@rm config/dev/azure-cloud-nuke.yaml
 
 .PHONY: cli-install
-cli-install: clusterawsadm clusterctl cloud-nuke envsubst yq awscli ## Install the necessary CLI tools for deployment, development and testing.
+cli-install: clusterawsadm clusterctl cloud-nuke envsubst yq awscli jq## Install the necessary CLI tools for deployment, development and testing.
+	@required_bins="clusterawsadm clusterctl cloud-nuke envsubst yq aws jq"; \
+	for bin in $$required_bins; do \
+		if ! command -v $$bin >/dev/null 2>&1; then \
+			$$bin ; \
```

Comment:
I did not get it, why do we need to call a missing binary before printing an error and exiting?

nit: the comment does not have space between targets and its beginning (`jq##...`)

Reply:
Ah! that was only for debugging.
Removing it now. Thanks.